pkg/operator/hive/hiveapi.go

```diff
@@ -28,18 +28,11 @@ func (r *ReconcileHiveConfig) deployHiveAPI(hLog log.FieldLogger, h *resource.He
 		return r.tearDownHiveAPI(hLog)
 	}
 
-	err := util.ApplyAsset(h, "config/apiserver/hiveapi-cluster-role-binding.yaml", hLog)
-	if err != nil {
-		return err
-	}
-
-	err = util.ApplyAsset(h, "config/apiserver/service.yaml", hLog)
-	if err != nil {
+	if err := util.ApplyAsset(h, "config/apiserver/service.yaml", hLog); err != nil {
```

Comment:
are we not applying the role binding from inside the hive operator b/c OLM is expected to perform this?

Reply:
Yes. The hive operator does not have the RBAC permissions that the apiserver needs so cannot give the apiserver those permissions.

Reply:
The OLM is doing this same task already for the hive admission service account.

Reply:
That is totally not what I remembered but absolutely correct.